apps/bfd-server/bfd-server-war/src/test/resources/openapi/posts.yaml

```
@@ -351,6 +349,232 @@ paths:
+                    example: "lt2023-05-01"
+                  startIndex:
+                    type: string
+                    description: |-
+                      When fetching a _Bundle Response_ using pagination, this URL parameter represents an offset
+                      (starting point) into the list of elements for the _Request_.
+                      It is optional and defaults to 1 if not supplied.
+                      A value 0 is not allowed and negative indices are not currently supported.
+
+                      Example:
+                      - `startIndex=100`
+                    example: 100
+                  type:
+                    type: string
+                    description: |-
+                      A list of one or more comma-separated claim types to be included in the request;
+                      within BFD, the claim types represent an _OR_ inclusion logic meaning any claims matching one of the specified
+                      claim types will be checked
+
+                      Supported Claim Type values:
+                        - `fiss`
+                        - `mcs`
+
+                      Examples:
+                        - `type=fiss,mcs`
+                        - `type=fiss`
+                    example: "fiss,mcs"
+          required: true
+        responses:
+          '200':
+            description: Success
+            content:
+              application/x-www-form-urlencoded:
+  /v2/fhir/ClaimResponse/_search:
+    post:
+      tags: [v2,ClaimResponse]
+      summary: 'search-type: Search for ClaimResponse instances'
+      description: This is a search type
+      requestBody:
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              type: object
+              properties:
+                mbi:
+                  type: string
+                  description: |-
+                    **NOTE: THIS IS A REQUIRED FIELD**
+                    
+                    Fetch _Beneficiary_ data using a FHIR _MBI_ identifier; an MBI
+                    represents the medicare benficiary ID, or as much of that
+                    defined by the [FHIR specification](https://terminology.hl7.org/NamingSystem-cmsMBI.html) as:
+                    
+                    ```
+                    A combination of upper case ASCII letters ('A'..'Z except for S, L, O, I, B, and Z.', numerals ('0'..'9'),
+                    '-', with a length limit of 11 characters.
+                    ```
+                    
+                    The following are all valid values for MBI:
+                      - `mbi=1EG4TE5MK73` (unhashed MBI)
+                      - `mbi=1S00E00HA26` (synthetic MBI)
+                  example: "1EG4TE5MK73"
+                _lastUpdated:
+                  type: string
+                  description: |-
+                    Only satisfy the Search if the Beneficiary's `last_updated` Date falls within a specified _DateRange_.
```

Here (and elsewhere), there's an issue with this text, but I'm not sure if it shouldn't be more generalized. It's easy to miss that this is specific to _this_ FHIR resource... maybe it shouldn't read "Claim's" rather, "this FHIR resource's"?

```suggestion
                    Only satisfy the Search if the Claim's `last_updated` Date falls within a specified _DateRange_.
```